.ci/kata_chronic.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Use chronic to mute the output of a command, unless it errors.
+# But, to prevent system timeouts from complete inactivity, periodically
+# output '.'s to keep the system looking alive...
+
+cmdLine=$@
+
+eval chronic ${cmdLine[@]} &
```

Comment:
This is prolly safer if quoted, which might also help the issue you were seeing with `&&` / `||`:

```suggestion
eval chronic "${cmdLine[@]}" &
```

Reply:
tried it already - didn't work :-(  I even put the `"`s outside the `chronic` as well to try and wrap it further for the eval - still didn't work.... but, sure, quoting it is probably the right thing to do (but it doesn't fix the `&&` and `||` issue afaict). I'll edit and push